data/scenario-17.json

```diff
@@ -0,0 +1,169 @@
+{
+  "locatorId": "c000000000ho",
+  "passport": {
+    "language": "en-gb",
+    "home": "http://www.bbc.co.uk/ontologies/passport/home/News",
+    "articleType": "analysis",
+    "genre": null
+  },
+  "publishedDateTime": "2018-06-04T00:00:00.000Z",
```

Comment:
Probably best not to have this at midnight for BST/GMT conversion, we can handle the BST logic in a more unique fixture.